snmp.yml

```diff
@@ -1565,3 +1565,946 @@ fortinet_fortigate:
           type: Integer32
         - labelname: fgWcWtpSessionWtpId
           type: OctetString
+          
+# MikroTik RouterOS Devices
+# There are a few undocumented MIBs in use in this, and a few MIBs that I was unable to test so they were not implemented
+# May require tweaking to meet your exact needs
+mikrotik_routeros:
+  walk:
+    - 1.3.6.1.4.1.14988.1.1
+  metrics:
+    - name: mtxrWlRtabAddr
+      oid: 	1.3.6.1.4.1.14988.1.1.1.2.1.1
+    - name: mtxrWlRtabRouterOSVersion
+      oid: 1.3.6.1.4.1.14988.1.1.1.2.1.10
+      indexes:
+        - labelname: mtxrWlRtabAddr
+          type: PhysAddress48
+    - name: mtxrWlRtabUptime
+      oid: 	1.3.6.1.4.1.14988.1.1.1.2.1.11
+      indexes:
+        - labelname: mtxrWlRtabAddr
+          type: PhysAddress48
+    - name: mtxrWlRtabSignalToNoise
+      oid: 1.3.6.1.4.1.14988.1.1.1.2.1.12
+      indexes:
+        - labelname: mtxrWlRtabAddr
+          type: PhysAddress48
+    - name: mtxrWlRtabIface
+      oid: 1.3.6.1.4.1.14988.1.1.1.2.1.2
+      indexes:
+        - labelname: mtxrWlRtabAddr
+          type: PhysAddress48
+    - name: mtxrWlRtabTxBytes
+      oid: 1.3.6.1.4.1.14988.1.1.1.2.1.4
+      indexes:
+        - labelname: mtxrWlRtabAddr
+          type: PhysAddress48
+    - name: mtxrWlRtabRxBytes
+      oid: 1.3.6.1.4.1.14988.1.1.1.2.1.5
+      indexes:
+        - labelname: mtxrWlRtabAddr
+          type: PhysAddress48
+    - name: mtxrWlRtabTxPackets
+      oid: 1.3.6.1.4.1.14988.1.1.1.2.1.6
+      indexes:
+        - labelname: mtxrWlRtabAddr
+          type: PhysAddress48
+    - name: mtxrWlRtabRxPackets
+      oid: 1.3.6.1.4.1.14988.1.1.1.2.1.7
+      indexes:
+        - labelname: mtxrWlRtabAddr
+          type: PhysAddress48
+    - name: mtxrWlRtabTxRate
+      oid: 1.3.6.1.4.1.14988.1.1.1.2.1.8
+      indexes:
+        - labelname: mtxrWlRtabAddr
+          type: PhysAddress48
+    - name: mtxrWlRtabRxRate
+      oid: 1.3.6.1.4.1.14988.1.1.1.2.1.9
+      indexes:
+        - labelname: mtxrWlRtabAddr
+          type: PhysAddress48
+
+    - name: mtxrWlApIndex
+      oid: 1.3.6.1.4.1.14988.1.1.1.3.1.1
+    - name: mtxrWlApOverallTxCCQ
+      oid: 1.3.6.1.4.1.14988.1.1.1.3.1.10
+      indexes:
+        - labelname: mtxrWlApIndex
+          type: Integer32
+      lookups:
```

Comment:
I don't think there's a way this lookup could be automatically calculated from the MIB, so this should be removed as once this is all auto-generated this won't be here.